### ISSUE TITLE
Add "Source Control Repositories" view in order to manage multiple git repos in one window like in VSCode

### ISSUE THREAD

**Issue by cbeanLeadr** [cursor/cursor#1759]
**Is your feature request related to a problem? Please describe.**
I want to manage multiple git repos in the same Cursor window so that I don't have to navigate between a bunch of different windows. However, the source control tab currently only lets me manage one git repo at a time as far as I can tell. This is very disruptive to me as someone that maintains multiple packages and microservices on a regular basis - it's pretty close to a non-starter to get me using Cursor on a regular basis since it doesn't work with my regular workflow.

**Describe the solution you'd like**
The ability to show the "Source Control Repositories" view as in VSCode (defaults to being hidden, so it only shows if you toggle it on). Then, also be able to close and reopen repositories from with this menu.

**Additional context**
Viewing multiple repositories at once:
![image](https://github.com/user-attachments/assets/a3c7e912-9210-4687-b7dc-bffdaa799986)

Toggling "Source Control Repositories" on:
![image](https://github.com/user-attachments/assets/8793089a-3126-4bd7-ab9c-a7585d466161)

Reopening closed repositories:
![image](https://github.com/user-attachments/assets/4e1717c1-1933-4d4f-b0ec-abef720dc76c)
![image](https://github.com/user-attachments/assets/9c91b4dc-61a4-4523-85ec-d4d0f15eda10)

**Comment by etjones-erisyon** [comment]
+1. Came here to make the same request

**Comment by etjones-erisyon** [comment]
@cbeanLeadr , I uncovered a multi-repository view in my Cursor SCM panel, but I had to dig a little. Here's a screenshot of the menu nesting, which lets you select which repositories to show: 

<img width="993" alt="Screenshot 2024-08-30 at 11 39 05 AM" src="https://github.com/user-attachments/assets/b0291f6c-5ddb-431f-83ec-5926bba958d5">

For reference, the options I chose to view multiple repos were:   Source control panel -> `...` -> "View & Sort" -> "Repositories" -> check/uncheck repositories to include.

This isn't identical to VSCode's interface, but it's close enough for my purposes. I'd move to close this issue.

**Comment by cbeanLeadr** [comment]
@etjones-erisyon Good catch. I'm unable to see that on my end though on Windows. I'm wondering if I'm missing something that would allow me to see this as well.
![image](https://github.com/user-attachments/assets/e22da38c-bffc-4cd0-ab99-f05b8a7b0ab9)

**Comment by faustogenga** [comment]
I had the same problem, for now i downloaded the GitHub Repository Manager by henriqueBruno to see al repos in cursor and you can download Gitlens to see the branches.
